scan.py

```diff
@@ -40,10 +40,13 @@
 from common import create_dir
 from common import get_timestamp
 
-from datetime import datetime
+from common import IS_AV_ENABLED
+from common import MIME_VALIDATION
+from common import MIME_VALIDATION_STATIC_VALID_LIST
+from common import MIME_VALIDATION_S3_CONTENT_TYPE
+from common import MIME_VALIDATION_STATIC
```

Comment:
Add to the `upgrade_common`, then it could be referenced as:

```suggestion
from upgrade_common import IS_AV_ENABLED, MIME_VALIDATION, MIME_VALIDATION_STATIC_VALID_LIST, MIME_VALIDATION_S3_CONTENT_TYPE, MIME_VALIDATION_STATIC
```

This does seem to go against this repo's code standards, but it will allow us to easily keep track our custom changes.